command/translatableerror/service_plan_not_found_error.go

```diff
@@ -1,20 +1,26 @@
 package translatableerror
```

Comment:
Some translatable errors have unit tests. This isn't a strict requirement, but can be a good way to test all the possibilities.

Reply:
Also, is this error actually used?

Reply:
Thanks! forgot to update them. 

It is I am afraid the one actually used for printing the message. The command parser converts to translatable error before printing to stderr. I could remove the mapping and the error altogether and then it would just use the one in actionerror package. But... this is used by v2 as well. Do you think we should remove it now that we are using branches instead of flags?